### PR TITLE
Wntr visualization improvements 

### PR DESCRIPTION
**Pull request by jjstickel**
- Enable showing solution information to the WNTR visualization. 
- Can merge the colorbar to the top of the page, rather than as a separate page. Ghostscript dependency is now removed for the Python-based visualization. Will add this capability to the WM graph visualization in another PR.
- Minor improvements to the WM graph visualization.

### CODE REVIEW

File: src/python/wntr_vis.py [hunk 1]
```diff
@@ -114,24 +123,87 @@ def build_graph(wn, time=1, wnsol=None):
...
         if clr[2] < 0.6:
             Gnx.nodes[node]["fontcolor"] = "white"
+    return
+
+
+def link_attrs(wn, Gnx):
+    """
+    Add/change link attributes suitable for visualization
+    """
+    # loop over controls to find pipes with shutoff valves
```
Comment (enzoliuyang94) [inline src/python/wntr_vis.py:133]: Alternatively, we can directly retrieve the shutoff valve names if their valve ids are known - all shutoff valves in the wntr object are named as 'valve'+'#VALVE ID#'.

**Review by enzoliuyang94 — APPROVED** [review]
Looks good to me. Happy holidays!

File: src/python/wntr_vis.py (hunk 1 above)
Comment (jjstickel) [reply src/python/wntr_vis.py:133]: > all shutoff valves in the wntr object are named as 'valve'+'#VALVE ID#'.
I don't think this is true. My test case has been `Richmond_standard.inp` in PowerWaterExamples. The shutoff valve identified by the `CONTROLS` section has the name `v1E`, but that is the name given in the inp file, and `valve` is not prepended to the name. The link type is `Pipe`. It seems that WNTR uses the `Valve` type for special valves like PRVs.

**Comment by jjstickel** [comment]
> Looks good to me. Happy holidays!

Thanks for the review!

**Pull request merged**